source/fuzz/fuzzer_pass.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/fuzz/fuzzer_pass.h"
+
+namespace spvtools {
+namespace fuzz {
+
+opt::IRContext* FuzzerPass::GetIRContext() const { return ir_context_; }
+
+FactManager* FuzzerPass::GetFactManager() const { return fact_manager_; }
+
+FuzzerContext* FuzzerPass::GetFuzzerContext() const { return fuzzer_context_; }
```

Comment:
These getters can be implementated in the .h file

Reply:
Done.